components/ai_chat/core/browser/model_service.cc

```diff
@@ -212,6 +214,13 @@ base::Value::Dict GetModelDict(mojom::ModelPtr model) {
       kCustomModelItemApiKey,
       EncryptAPIKey(model->options->get_custom_model_options()->api_key));
   model_dict.Set(kCustomModelItemKey, model->key);
+
+  std::optional<uint32_t> context_size =
+      model->options->get_custom_model_options()->context_size;
+  if (context_size.has_value()) {
```

Comment:
super optional but I think optionals implicitly convert to booleans so you can do:

up to you, its just preference really

```suggestion
  if (context_size) {
```

Reply:
I'm a fan of the explicitness here. That said, as I become more comfortable with working in a strongly-typed language, I may find that `.has_value` is too verbose 🙂